__tests__/integration/previewSaveRDF.test.js

```diff
@@ -24,8 +24,8 @@ describe('Expanding a resource property in a property panel', () => {
     await pupExpect(page).toMatch('If this looks good, then click Save and Publish')
   })
 
-  it('presents a choice of group to save to', async () => {
-    await pupExpect(page).toClick('button', { text: 'Save & Publish' })
+  it('presents a choice of group to save to from the RDFModal', async () => {
+    await pupExpect(page).toClick('#modal-save', { text: 'Save & Publish' })
```

Comment:
`#modal-save` is needed in order for the integration test to distinguish between `Save & Publish` buttons.